src/test/java/teammates/test/cases/storage/FeedbackResponseCommentsDbTest.java

```diff
@@ -206,16 +204,13 @@ private void testGetFeedbackResponseCommentForGiver() {
 
     private void testGetFeedbackResponseCommentForResponse() {
         String responseId = "1%student1InCourse1@gmail.tmt%student1InCourse1@gmail.tmt";
-        ArrayList<FeedbackResponseCommentAttributes> frcasExpected =
-                new ArrayList<FeedbackResponseCommentAttributes>();
+        ArrayList<FeedbackResponseCommentAttributes> frcasExpected = new ArrayList<>();
         frcasExpected.add(frcaData);
 
         ______TS("typical success case");
 
-        List<FeedbackResponseCommentAttributes> frcas =
-                frcDb.getFeedbackResponseCommentsForResponse(responseId);
-        verifyListsContainSameResponseCommentAttributes(
-                new ArrayList<FeedbackResponseCommentAttributes>(frcasExpected), frcas);
+        List<FeedbackResponseCommentAttributes> frcas = frcDb.getFeedbackResponseCommentsForResponse(responseId);
```

Comment:
This is unrelated, but I'm okay with it since it's in close proximity to and of the same nature as the change in the below line (which is related).